src/compiler/scala/tools/nsc/typechecker/RefChecks.scala

```diff
@@ -440,10 +440,13 @@ abstract class RefChecks extends Transform {
               // Don't bother users with deprecations caused by classes they inherit.
               // Only warn for the pair that has one leg in `clazz`.
               if (clazz == memberClass) checkOverrideDeprecated()
-              if (settings.warnNullaryOverride) {
-                def javaDetermined(sym: Symbol) = sym.isJavaDefined || isUniversalMember(sym)
-                if (other.paramss.isEmpty && !member.paramss.isEmpty && !javaDetermined(member) && !member.overrides.exists(javaDetermined))
-                  refchecksWarning(member.pos, "non-nullary method overrides nullary method", WarningCategory.LintNullaryOverride)
+              def javaDetermined(sym: Symbol) = sym.isJavaDefined || isUniversalMember(sym)
+              if (other.paramss.isEmpty && !member.paramss.isEmpty && !javaDetermined(member) && !member.overrides.exists(javaDetermined)) {
+                val msg = "non-nullary method overrides nullary method"
```

Comment:
I think we should avoid the "nullary" term, it's too technical

Reply:
This is the ancient formulation that precedes our distinctions for `paramss.isEmpty` and `paramss.head.isEmpty`. I know you've advocated for "empty parameter list" or was it "empty args", elsewhere. "Method requiring parens, method disallowing parens." Or "method such as scorneth the parenthesis."

Reply:
Changed.  Re-comment on the new version if you think it needs tweaking.